File: oop_grund/Inlämning/Airport/Airport/Human.cs
namespace Airport
{
    public abstract class Human
    {
        private string name;
        private int age;
        
        protected Human(string name, int age)
        {
            this.name = name;
            this.age = age;
        }


        public string GetName()
        {
            return name;
        }

        public int GetAge()
        {
            return age;
        }
    }
}

File: oop_grund/Inlämning/Airport/src/main/java/with/william/airport/human/Human.java
package with.william.airport.human;

import with.william.airport.util.Country;

import java.time.LocalDate;

public abstract class Human {

    private LocalDate dateOfBirth; //added date of birth.
    private Country country; //added country
    private float balance = 800; // added money

    private String name;

    protected Human(String fullName, Country country, LocalDate dateOfBirth) {
        this.name = fullName;
        this.country = country;
        this.dateOfBirth = dateOfBirth;

    }

    public LocalDate getDateOfBirth() {
        return dateOfBirth;
    }

    public Country getCountry() {
        return country;
    }

    public String getName() {
        return name;
    }

    public float getBalance() {
        return balance;
    }

    public void setBalance(float balance) {
        this.balance = balance;
    }
}
